src/components/data-display/tables/TableView.tsx

```diff
@@ -102,8 +104,8 @@ const TableView = <T, S>({
   return (
     <>
       <div className={`overflow-x-auto scrollbar ${tableWrapperClassName}`}>
-        <table className={'w-full table-auto' + ' ' + tableClass}>
-          <thead className={`text-[0.875rem] text-grey ${headerClass}`}>
+        <table className={`${tableClass} w-full table-auto`}>
```

Comment:
This changes the order of classes from appending to prepending. Need to be careful here and the line below. Would suggest reordering to make it come at end to match prior behavior to limit scope of change.

Reply:
yup this was a deliberate change. Prepending ensures overridability of the default classes but will still add them if not conflicting.

Reply:
Hmm, seems like ordering doesn't determine overridability. We may want to look at using tailwind-merge at some point. (In another PR...)